.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
```

Comment:
A minor thing: I think this should match the version in `.nvmrc` (now v16.14.0). Should we put to `.nvmrc` also only the major version 🤔

Reply:
I'm actually surprised that setup-node doesn't have an option to read the nvmrc itself... I'll just double check. We could use the major version in both if that doesn't work out, I doubt it makes a difference. ✅

Reply:
Actually I now found that one could have a "read nvmrc step" that would do just that - it does look a bit scary but does the job 😅 Let me know what you think, if this is better or worse?